content/docs/command-reference/queue/kill.md

```diff
@@ -0,0 +1,31 @@
+## queue kill
```

Comment:
Should we add a note somewhere that to kill all experiments, use `queue stop --kill`?